packages/suite/src/components/suite/ToastNotification.tsx

```diff
@@ -10,44 +10,44 @@ import NotificationRenderer, {
     NotificationViewProps,
 } from '@suite-components/NotificationRenderer';
 
-const Wrapper = styled.div<Pick<NotificationViewProps, 'variant'>>`
+const Wrapper = styled.div<Pick<NotificationViewProps, 'variant'> & { isTall: boolean }>`
     display: flex;
-    align-items: center;
+    align-items: ${({ isTall }) => (isTall ? 'start' : 'center')};
     font-size: ${variables.FONT_SIZE.SMALL};
     height: 100%;
-    padding: 6px 12px;
-    border-left: 4px solid ${props => getVariantColor(props.variant)};
+    padding: ${({ isTall }) => (isTall ? '16px 16px 12px 12px' : '16px')};
```

Comment:
```suggestion
    padding: ${({ isTall }) => (isTall ? '12px 16px' : '16px')};
```
Do you really want to change the left padding?

Reply:
Yep, since there is a like on the left, which is white in the case of the URI notification, red for errors, etc. If I keep the the left padding the same, the distance from the left edge to the content would actually visibly be larger than on the right :D